explain-indexes.md

```diff
@@ -0,0 +1,343 @@
+| └─TableRowIDScan_9(Probe)     | 5.67    | cop[tikv] | table:t1                       | keep order:false                  |
++-------------------------------+---------+-----------+--------------------------------+-----------------------------------+
+3 rows in set (0.00 sec)
+
++-------------------------------+---------+-----------+--------------------------------+-----------------------------------------------------+
+| id                            | estRows | task      | access object                  | operator info                                       |
++-------------------------------+---------+-----------+--------------------------------+-----------------------------------------------------+
+| IndexLookUp_10                | 4.00    | root      |                                |                                                     |
+| ├─IndexRangeScan_8(Build)     | 4.00    | cop[tikv] | table:t1, index:intkey(intkey) | range:[29,29], [98,98], [123,123], keep order:false |
+| └─TableRowIDScan_9(Probe)     | 4.00    | cop[tikv] | table:t1                       | keep order:false                                    |
++-------------------------------+---------+-----------+--------------------------------+-----------------------------------------------------+
+3 rows in set (0.00 sec)
+
++-------------------------------+---------+-----------+--------------------------------+----------------------------------+
+| id                            | estRows | task      | access object                  | operator info                    |
++-------------------------------+---------+-----------+--------------------------------+----------------------------------+
+| IndexLookUp_10                | 6.00    | root      |                                |                                  |
+| ├─IndexRangeScan_8(Build)     | 6.00    | cop[tikv] | table:t1, index:intkey(intkey) | range:[99,103], keep order:false |
+| └─TableRowIDScan_9(Probe)     | 6.00    | cop[tikv] | table:t1                       | keep order:false                 |
++-------------------------------+---------+-----------+--------------------------------+----------------------------------+
+3 rows in set (0.00 sec)
+```
+
+The `IndexLookup` operator has two child nodes:
+
+* The `├─IndexRangeScan_8(Build)` performs a range scan on the `intkey` index and retries the value of the internal `RowID`, which for this table is also the `PRIMARY KEY`.
+* The `└─TableRowIDScan_9(Probe)` operator then retrieves the full row from the table data.
+
+Because an `IndexLookup` requires two steps, the SQL Optimizer may elect to use the `TableFullScan` operator in scenarios where a large number of rows match. The optimizer uses [statistics](/statistics.md) in order to make these decisions. In the following example, a large number of rows match the condition of `intkey > 100`, and a `TableFullScan` is chosen:
```

Comment:
```suggestion
Because an `IndexLookup` task requires two steps, the SQL Optimizer might choose the `TableFullScan` operator based on [statistics](/statistics.md) in scenarios where a large number of rows match. In the following example, a large number of rows match the condition of `intkey > 100`, and a `TableFullScan` is chosen:
```